src/sagas/base-data.js

```diff
@@ -108,16 +108,61 @@ export function* fetchAllVnicProfiles (action) {
   }
 }
 
+/**
+ * Fetch the user's ovirt known groups.  To do so, we need to fetch and cross reference the
+ * users's domain groups with the ovirt known groups.  Users can have more domain groups
+ * than ovirt groups.  The cross reference will turn domain groups membership into ovirt
+ * group uuids.  Permission checks require ovirt group uuids, not the domain entity ids.
+ */
```

Comment:
> Users can have more domainGroups then ovirt groups
>

The opposite situation happened during my tests:
1. `/api/groups` returns 3 entries i.e. `Everyone`, `ovirt-administrator`
2. `/api/users/{id}/groups` returns empty array

Is this behavior correct?

Reply:
Well, the `/api/groups` will return all of the groups that the user has access to see.  So yeah, if your user is an admin user, they can see all of the groups but don't need to actually belong to any.  I'll update the comment a bit.